tools/nlp-toolkit/src/NlpToolkit.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+	<div class="NlpToolkit">
+
+		<tool-markdown :markdown="`
+This is a tool to to perform NLP operations using the library [compromise](https://www.npmjs.com/package/compromise).
+Enter text below: 		`"/>
```

Comment:
spacing looks funny here, probably since we have the multi-line property `markdown`. Let's the tag open and close like this: https://github.com/holistic-web/holistic-toolbox/blob/master/tools/number-converter/src/NumberConverter.vue#L4-L9

Reply:
Done.